config/jsdoc/api/plugins/doc.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * Define an @doc tag
+ */
+var conf = env.conf.stability;
+var defaultLevels = ["deprecated","experimental","unstable","stable","frozen","locked"];
```

Comment:
It is, but I don't think we need to handle that for things that are not part of the API.